Dockerfile.dev

```diff
@@ -6,6 +6,8 @@ RUN \
     curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
     && apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        # Additional library needed by fjaraskupan tests and accordingly by VScode Tests Discovery
```

Comment:
```suggestion
```

Reply:
Either this or mention bleak. Bleak probably should delay this evaluation.